src/promptflow-tools/CHANGELOG.md

```diff
@@ -1,7 +1,47 @@
 # Release History
 
+## 1.4.0 (Upcoming)
+
+### Features Added
+- Enable token based auth in azure openai connection for tools.
+- Add "seed" to LLM and GPT-Vision tool inputs.
+
+## 1.3.0 (2024.03.01)
+
+### Improvements
+- Disable openai built-in retry mechanism for better debuggability and real-time status updates.
+- Refine the retry-after interval for openai retry error.
+
+## 1.2.0 (2024.02.07)
+
+### Features Added
+- Support tool "Azure OpenAI GPT-4 Turbo with Vision" to auto list "deployment_name".
+
+### Improvements
+- Match the promptflow-tools setup to promptflow setup.
+
+## 1.1.0 (2024.01.23)
+
+### Features Added
+- Use ruamel.yaml instead of pyyaml in promptflow.
+
+## 1.0.3 (2024.01.08)
+
+### Features Added
```

Comment:
missing 1.0.2 ?

Reply:
We had two packages(1.0.2 and 1.0.3) released on the same day, so merge them.

![image](https://github.com/microsoft/promptflow/assets/49483542/4a31c40e-014b-4dee-b947-f8b4970053f7)


> missing 1.0.2 ?